content/participate/code.adoc

```diff
@@ -10,7 +10,7 @@ tags:
 Jenkins project includes a lot of code, and we invite everyone to contribute to the project.
 There is a diverse set of programming languages used in Jenkins,
 including but not limited to: Java, JavaScript, Groovy, Golang, Ruby, Shell scripts.
-And, since Jenkins is an automation server with hundreds of plugins, there is a huge number of technologies involved.
+And, since Jenkins is an automation server with roughly 1,800 plugins, there is a huge number of technologies involved.
```

Comment:
the fewer places mention the precise number the better, I'd suggest to keep hundreds here.

Reply:
"hundreds" is wrong because there are not "hundreds" of plugins any longer. 1,800 aligns with our official press page and various other official sources we're using that in.
This change solely aligns with what's already in place.

I see no reason in retaining an outdated and false figure.

Reply:
No strong opinion, but I agree with @zbynek that it is better to keep a generic phrase here

Reply:
it's just that 1800 can soon become 1900 or 1700 (e.g. if plugins incompatible with Java 11 get removed...) and nobody will remember to update this page. Anyway, just a nitpick, feel free to merge if you disagree.

cf https://github.com/jenkins-infra/plugin-site/pull/679

Reply:
> it's just that 1800 can soon become 1900 or 1700

Updating 1800, 1900 or 1700 in all places is much more likely and easier than searching for an undefined amount nobody knows of, imo 🤷🏻